velocitas_lib/src/velocitas_lib/services.py

```diff
@@ -0,0 +1,130 @@
+import json
+from pathlib import Path
+from typing import Dict, List, NamedTuple, Optional
+
+from velocitas_lib import get_package_path, get_workspace_dir, require_env
+from velocitas_lib.variables import ProjectVariables
+
+
+class ServiceSpecConfig(NamedTuple):
+    image: str
+    is_enabled: bool = True
+    env_vars: dict[str, Optional[str]] = dict()
+    no_dapr: bool = False
+    args: List[str] = list()
+    ports: List[str] = list()
+    port_forwards: List[str] = list()
+    mounts: List[str] = list()
+    patterns: List[str] = list()
+
+
+class Service(NamedTuple):
+    id: str
+    config: ServiceSpecConfig
+
+
+def parse_service_config(service_spec_config: Dict) -> ServiceSpecConfig:
+    """Parse service spec configuration and return it as an named tuple.
+
+    Args:
+        service_spec_config: The specificon of the services from config file.
+    """
+
+    is_enabled = True
+    no_dapr = False
```

Comment:
I would prefer
```suggestion
    dapr = True
```
but this is not that important, just personal opinion

Reply:
Agreed on `use_dapr`